bin/install.exec

```diff
@@ -15,5 +15,5 @@ if [[ TARGET_INSTALL_REMOTE -eq 1 ]]; then
 
 	exec ${args[@]}
 else
-	exec sh -c "$@"
+	exec su -c "$@"
```

Comment:
Not sure about this. It should be ok, but there may be some edge cases of projects that assume their `install.exec` runs as the same user.